monitor/dbt_project/macros/get_new_test_alerts.sql

```diff
@@ -4,6 +4,33 @@
         SELECT * FROM {{ ref('alerts_tests') }}
         WHERE alert_sent = FALSE and detected_at >= {{ get_alerts_time_limit(days_back) }}
     {% endset %}
+    {% set select_nodes_meta_query %}
```

Comment:
I think it makes more sense either to join with model_id and bring the meta or add it to the alert in the package - wdyt?

Reply:
All options are good IMO.
In the current situation where changes were about to occur in the package, I though that this is the most suitable solution.